sequel/lib/utils.js

```diff
@@ -30,6 +30,25 @@ utils.object.hasOwnProperty = function(obj, prop) {
 
 
 /**
+ * String literal helper function
+ *
+ * Pass in a value and have it wrapped in a string literal.
+ * Saves having self.escapeCharacter littered everywhere
+ *
+ * @param {String} val
+ * @param {String} escapeCharacter
+ * @return {String}
+ * @api public
+ *
+ */
+
+utils.stringLiteral = function stringLiteral(val, escapeCharacter) {
+  var regex = new RegExp(escapeCharacter, 'g');
+  return escapeCharacter+val.replace(regex, '\\'+escapeCharacter)+escapeCharacter;
```

Comment:
I'm concerned about this approach.  Consider the javascript string literal `"bl\'ue"` turning into `'bl\\\'ue'` in SQL.

Reply:
@devinivy are you comfortable if I import [this](https://www.npmjs.com/package/sqlstring) to do the sql escaping?